.gitignore

```diff
@@ -15,3 +15,6 @@ project/plugins/project/
 # Scala-IDE specific
 .scala_dependencies
 .worksheet
+
+# IntelliJ specific
+.idea
```

Comment:
OK by me.

Did you know you can also add such ignores to your system globally? A nice collection can be found at https://github.com/github/gitignore/tree/master/Global